builtin/providers/azurerm/resource_arm_virtual_machine_scale_set.go

```diff
@@ -74,6 +74,13 @@ func resourceArmVirtualMachineScaleSet() *schema.Resource {
 				Optional: true,
 			},
 
+			"single_placement_group": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+				ForceNew: true,
```

Comment:
From https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-placement-groups: 
> You can change a scale set from supporting a single placement group only (the default behavior) to a supporting multiple placement groups, but you cannot convert the other way around. Therefore make sure you understand the properties of large scale sets before converting. In particular, make sure you do not need layer-4 load balancing with the Azure Load Balancer.

This isn't the only place we've seen this.  Is there a better way to handle this?  I figured forcing a new resource when not needed was better than having it completely fail when it is needed?